mdm-connector.iml

```diff
@@ -29,10 +29,10 @@
     <orderEntry type="library" name="Maven: com.squareup.okio:okio:1.15.0" level="project" />
     <orderEntry type="library" name="Maven: org.bouncycastle:bcprov-jdk15on:1.65" level="project" />
     <orderEntry type="library" name="Maven: de.fraunhofer.iais.eis.ids:connector-commons:4.0.2-SNAPSHOT" level="project" />
-    <orderEntry type="library" name="Maven: de.fraunhofer.iais.eis.ids:negotiation-library:0.0.3-SNAPSHOT" level="project" />
```

Comment:
IML is Intellij related stuff. Please exclude

Reply:
Done